README.md

```diff
@@ -57,120 +60,81 @@ attach an event listener after the `deviceready` event fires.
 
 ### Example
 
-    window.addEventListener("batterystatus", onBatteryStatus, false);
+    window.addEventListener("batterylow", onBatteryLow, false);
 
-    function onBatteryStatus(info) {
-        // Handle the online event
-        console.log("Level: " + info.level + " isPlugged: " + info.isPlugged);
+    function onBatteryLow(status) {
+        alert("Battery Level Low " + status.level + "%");
     }
 
-## batterycritical
-
-The event fires when the percentage of battery charge has reached the
-critical battery threshold. The value is device-specific.
-
-The `batterycritical` handler is passed an object that contains two
-properties:
-
-- __level__: The percentage of battery charge (0-100). _(Number)_
-
-- __isPlugged__: A boolean that indicates whether the device is plugged in. _(Boolean)_
-
-Applications typically should use `window.addEventListener` to attach
-an event listener once the `deviceready` event fires.
-
 ### Supported Platforms
 
 - Amazon Fire OS
 - iOS
 - Android
 - BlackBerry 10
-- Tizen
 - Firefox OS
 - Windows (Windows Phone 8.1 only)
-- Browser
+- Browsers
+    - Chrome
+    - Firefox
+    - Opera
 
-### Windows Quirks
+### Quirks: Windows &amp; Windows Phone 8.1
 
-Windows Phone 8.1 will fire `batterycritical` event regardless of plugged state as it is not supported.
+The `batterylow` event fires on Windows Phone 8.1 irrespective of whether the device is plugged in or not. This happens because the OS does not provide an API to detect whether the device is plugged in.
+
+## batterycritical event
+
+Fires when the battery charge percentage reaches the critical charge threshold. This threshold value is device-specific. Returns an [object][status_object] containing battery status.
 
 ### Example
 
     window.addEventListener("batterycritical", onBatteryCritical, false);
 
-    function onBatteryCritical(info) {
-        // Handle the battery critical event
-        alert("Battery Level Critical " + info.level + "%\nRecharge Soon!");
+    function onBatteryCritical(status) {
+        alert("Battery Level Critical " + status.level + "%\nRecharge Soon!");
     }
 
-### Browser Quirks
-
-Supported browsers are Chrome, Firefox and Opera. 
-
-## batterylow
-
-The event fires when the percentage of battery charge has reached the
-low battery threshold, device-specific value.
-
-The `batterylow` handler is passed an object that contains two
-properties:
-
-- __level__: The percentage of battery charge (0-100). _(Number)_
-
-- __isPlugged__: A boolean that indicates whether the device is plugged in. _(Boolean)_
-
-Applications typically should use `window.addEventListener` to
-attach an event listener once the `deviceready` event fires.
-
 ### Supported Platforms
 
 - Amazon Fire OS
 - iOS
 - Android
 - BlackBerry 10
-- Tizen
 - Firefox OS
 - Windows (Windows Phone 8.1 only)
-- Browser
-
-### Windows Quirks
+- Browsers
```

Comment:
Browser is the name of the platform. For the supported ones, maybe just list them in parentheses like windows does above